### PR TITLE
AC Test Fix - LRAC-10604 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://testray.liferay.com/home/-/testray/case_results?tab=runs&testrayBuildId=1613321028 (run)
https://issues.liferay.com/browse/LRAC-10604 (ticket)

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase [hunk 1]
```diff
@@ -273,7 +273,7 @@ definition {
 	@priority = "5"
 	test CheckIfKnownIndividualsDontDisappearAfterDeleteStaticSegment {
 		task ("Connect the DXP to AC") {
-			ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+			var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase:276]: Hey @AfonsoHSantos, this way is no longer needed.

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**